<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Seppuku</class>
 <widget class="QWidget" name="Seppuku">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>671</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>671</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>930</width>
        <height>651</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>dRonin Seppuku Hardware Configuration</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QGraphicsView" name="boardDiagram">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>288</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: transparent</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="interactive">
             <bool>false</bool>
            </property>
            <property name="renderHints">
             <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2" colspan="2">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>113</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <property name="labelAlignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <item row="1" column="0">
               <widget class="QLabel" name="lblUart3">
                <property name="styleSheet">
                 <string notr="true">background-color: #ab37c8;
color: white;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
                </property>
                <property name="text">
                 <string>UART3:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="cbUart3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="objrelation" stdset="0">
                 <stringlist notr="true">
                  <string>objname:HwSeppuku</string>
                  <string>fieldname:Uart3</string>
                  <string>haslimits:yes</string>
                 </stringlist>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lblUart6">
                <property name="styleSheet">
                 <string notr="true">background-color: #ff6600;
color: white;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
                </property>
                <property name="text">
                 <string>UART6:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="lblUart1">
                <property name="styleSheet">
                 <string notr="true">background-color: #de8787;
color: white;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
                </property>
                <property name="text">
                 <string>UART1:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="cbUart6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="objrelation" stdset="0">
                 <stringlist notr="true">
                  <string>objname:HwSeppuku</string>
                  <string>fieldname:Uart6</string>
                  <string>haslimits:yes</string>
                 </stringlist>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="cbUart1">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="objrelation" stdset="0">
                 <stringlist notr="true">
                  <string>objname:HwSeppuku</string>
                  <string>fieldname:Uart1</string>
                  <string>haslimits:yes</string>
                 </stringlist>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="8" column="1" colspan="3">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" colspan="3">
           <layout class="QFormLayout" name="formLayout_2">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lblOutputs">
              <property name="styleSheet">
               <string notr="true">background-color: #000093;
color: white;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
              </property>
              <property name="text">
               <string>Outputs:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cbOutputs">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="objrelation" stdset="0">
               <stringlist notr="true">
                <string>objname:HwSeppuku</string>
                <string>fieldname:OutputConfiguration</string>
                <string>haslimits:yes</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblUart4">
              <property name="styleSheet">
               <string notr="true">background-color: #000093;
color: white;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
              </property>
              <property name="text">
               <string>UART4:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cbUart4">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="objrelation" stdset="0">
               <stringlist notr="true">
                <string>objname:HwSeppuku</string>
                <string>fieldname:Uart4</string>
                <string>haslimits:yes</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblDsmMode">
              <property name="styleSheet">
               <string notr="true">background-color: #008000;
color: white;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
              </property>
              <property name="text">
               <string>DSM Mode:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="cbDsmMode">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="objrelation" stdset="0">
               <stringlist notr="true">
                <string>objname:HwSeppuku</string>
                <string>fieldname:DSMxMode</string>
                <string>haslimits:yes</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lblMag">
              <property name="styleSheet">
               <string notr="true">background-color: #ab37c8;
color: white;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
              </property>
              <property name="text">
               <string>Magnetometer:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="cbMag">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="objrelation" stdset="0">
               <stringlist notr="true">
                <string>objname:HwSeppuku</string>
                <string>fieldname:Magnetometer</string>
                <string>haslimits:yes</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="lblExtMagOrientation">
              <property name="styleSheet">
               <string notr="true">background-color: #ab37c8;
color: white;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
              </property>
              <property name="text">
               <string>Ext. Mag. Orientation:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="cbExtMagOrientation">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="objrelation" stdset="0">
               <stringlist notr="true">
                <string>objname:HwSeppuku</string>
                <string>fieldname:ExtMagOrientation</string>
                <string>haslimits:yes</string>
               </stringlist>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="3" rowspan="2" colspan="2">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>80</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout">
              <property name="labelAlignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="lblRcvrPort">
                <property name="styleSheet">
                 <string notr="true">background-color: #008000;
color: white;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
                </property>
                <property name="text">
                 <string>Receiver:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cbRcvrPort">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="objrelation" stdset="0">
                 <stringlist notr="true">
                  <string>objname:HwSeppuku</string>
                  <string>fieldname:RcvrPort</string>
                  <string>haslimits:yes</string>
                 </stringlist>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>67</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lblUsbHid">
                <property name="styleSheet">
                 <string notr="true">background-color: #cccccc;
color: black;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
                </property>
                <property name="text">
                 <string>USB HID:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="cbUsbHid">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="objrelation" stdset="0">
                 <stringlist notr="true">
                  <string>objname:HwSeppuku</string>
                  <string>fieldname:USB_HIDPort</string>
                  <string>haslimits:yes</string>
                 </stringlist>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="lblUsbVcp">
                <property name="styleSheet">
                 <string notr="true">background-color: #cccccc;
color: black;
border-radius: 5px;
padding: 3px;
margin: 2px 0 2px;</string>
                </property>
                <property name="text">
                 <string>USB VCP:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="cbUsbVcp">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>180</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="objrelation" stdset="0">
                 <stringlist notr="true">
                  <string>objname:HwSeppuku</string>
                  <string>fieldname:USB_VCPPort</string>
                  <string>haslimits:yes</string>
                 </stringlist>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbMessages">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel[severity=warning] {
	color: black;
}

QLabel[severity=error] {
	color: red;
}</string>
         </property>
         <property name="title">
          <string>Messages</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnConnections">
           <property name="text">
            <string>Connection Diagram</string>
           </property>
           <property name="objrelation" stdset="0">
            <stringlist>
             <string>button:connectiondiagram</string>
            </stringlist>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnApply">
           <property name="text">
            <string>Apply</string>
           </property>
           <property name="objrelation" stdset="0">
            <stringlist notr="true">
             <string>button:apply</string>
            </stringlist>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSave">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="objrelation" stdset="0">
            <stringlist notr="true">
             <string>button:save</string>
            </stringlist>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnReboot">
           <property name="text">
            <string>Reboot</string>
           </property>
           <property name="objrelation" stdset="0">
            <stringlist notr="true">
             <string>button:reboot</string>
            </stringlist>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
